dropzone/index.d.ts

```diff
@@ -178,7 +178,7 @@ declare class Dropzone {
 
 	on(eventName:string, callback:(...args:any[]) => void):Dropzone;
 
-	off(eventName:string):Dropzone;
+	off(eventName?:string, callback?:(...args:any[]) => void):Dropzone;
```

Comment:
Technically it looks like there should be two overloads:

```ts
off(): Dropzone;
off(eventName: string, callback?: (...args:any[]) => void): Dropzone;
```

Based on [this comment from the source](https://github.com/enyo/dropzone/blob/4e20bd4c508179997b4b172eb66e927f9c0c8564/src/dropzone.coffee#L56-L58).